apps/backend/src/drink-action/dto/create-drink-action.dto.ts

```diff
@@ -0,0 +1,4 @@
+import { OmitType } from '@nestjs/swagger';
```

Comment:
You can fix this by running `yarn lint:fix`. If you have the ESLint extension in VSCode, it should show the error and may even fix it when saving the file.

Reply:
![image](https://github.com/kir-dev/BeAlcoholic/assets/134506897/fa178f4e-aa94-46c2-ab4c-1a025f37e7fd)
i dont know whats the problem, i have eslint...

Reply:
I have no idea why it doesn't find the apps directory. Maybe try running it manually: `npx eslint apps/* --ext .ts,.tsx --fix`

Reply:
can you try running from a git bash terminal? It's possible that powershell doesn't like forward slashes

Reply:
i tried both and it still says the same error message:l

Reply:
The command was invalid on windows, I'll push to master to fix this. You can do the linting with `yarn eslint apps --ext .ts,.tsx --fix` until then.